contracts/Internal.sol

```diff
@@ -39,8 +38,7 @@ contract Internal is Organized {
     /* Storage */
 
     /**
-     * Variable is defined private to highlight that even derived contracts
-     * are not able to modify the internal actors.
+     * It denotes the address is registered as internal actor or not.
```

Comment:
I suggest below for more clarity:

/* Mapping stores addresses which are registered as internal actor. */